### ISSUE TITLE
Webresource.asmx should be a secondary check

### ISSUE THREAD

**Issue by jcran** [intrigueio/intrigue-ident#23]
As surfaced in the comments on #20, Webresource.asmx is being requested as a first-round resource. Figure out which check is requesting it, and make it a follow-on check to ASP.net. 

```jcran ident master [20200317]$ ./uti./util/ident.rb -d -u [URL]
Checking URL: http://server.amconstruct1.com:8080
Getting http://server.amconstruct1.com:8080
Getting http://server.amconstruct1.com:8080/api
Getting http://server.amconstruct1.com:8080/doesntexist-123
Getting http://server.amconstruct1.com:8080/WebResource.asmx
Getting http://server.amconstruct1.com:8080/admin
Getting http://server.amconstruct1.com:8080/error.json
Ran 576 checks against base URL
Fingerprint:
 - Nginx Nginx   - Nginx (no version) (CPE: cpe:2.3:a:nginx:nginx::) (Tags: ["Web Server"]) (Hide: false)
```

**Comment by jcran** [comment]
cc @jen140 might be a good one for you to knock out.

**Comment by jen140** [comment]
Will be fixed with #24
